tests/test_lo.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+from rdkit.Chem import AllChem
+import datamol as dm
+from rdkit import Chem, DataStructs
+
+from splito.lohi import LoSplitter
+
+
+def test_lo():
+    data = dm.solubility()
+    data = data.drop_duplicates(["smiles"])
+
+    smiles = []
+    val = []
+
+    for idx in range(len(data)):
+        if AllChem.MolFromSmiles(data.iloc[idx]["smiles"]):
+            smiles.append(data.iloc[idx]["smiles"])
+            val.append(data.iloc[idx]["SOL"])
+
+    smiles = np.array(smiles)
+    val = np.array(val)
+
+    # deafult parameters
+    splitter = LoSplitter()
+    train_idx, clusters_idx = splitter.split(smiles, val)
+
+    for cluster in clusters_idx:
+        one_cluster_check(train_idx, cluster, smiles, 0.4, 5, val, 0.60)
+
+    # different parameters
+    splitter = LoSplitter(threshold=0.6, min_cluster_size=7, std_threshold=0.8)
+    train_idx, clusters_idx = splitter.split(smiles, val)
+
+    for cluster in clusters_idx:
+        one_cluster_check(train_idx, cluster, smiles, 0.6, 7, val, 0.80)
```

Comment:
In #12, I expanded the set of test fixtures defined in `conftest.py`. 

After merging your branch with main, you will be able to e.g. simplify this test case like: 

```suggestion
def test_lo(test_dataset_smiles, test_dataset_targets):

    # deafult parameters
    splitter = LoSplitter()
    train_idx, clusters_idx = splitter.split(test_dataset_smiles, test_dataset_targets)

    for cluster in clusters_idx:
        one_cluster_check(train_idx, cluster, test_dataset_smiles, 0.4, 5, test_dataset_targets, 0.60)

    # different parameters
    splitter = LoSplitter(threshold=0.6, min_cluster_size=7, std_threshold=0.8)
    train_idx, clusters_idx = splitter.split(test_dataset_smiles, test_dataset_targets)

    for cluster in clusters_idx:
        one_cluster_check(train_idx, cluster, test_dataset_smiles, 0.6, 7, test_dataset_targets, 0.80)
```

Reply:
Thanks, done